defender/gasless-nft-minting/README.md

```diff
@@ -0,0 +1,168 @@
+# Gasless NFT Minter
+
+This code will allow users to deploy an [ERC-2771-compatible](https://docs.openzeppelin.com/contracts/4.x/api/metatx#ERC2771Context) [ERC-1155 NFT](https://docs.openzeppelin.com/contracts/4.x/erc1155) contract and then use one Autotask to sign mint requests, and another Autotask to relay the request to a [trusted forwarder](https://docs.openzeppelin.com/contracts/4.x/api/metatx#MinimalForwarder) which will send the request to the NFT contract to mint an NFT for the user.
+
+## Setup
+
+This section will help you set up the following:
+- Acquire Defender and Relay keys
+- Add funds to the Relay
+- Deploy contracts
+  - `yarn deploy contract --contract-name MinimalForwarder`
+  - `yarn deploy contract --contract-name DemoNFT "0xMINIMAL_FORWARDER_ADDRESS"`
+- Update `config.dev.yml` with the 2 deployed contract addresses
+- Deploy the Autotasks with `serverless deploy`
+- Connect Relays to deployed Autotasks
+  - Contract Deployer relay must be connected to the signing Autotask
+  - Any Relayer with funds can be connected to the Relay Autotask
+- Copy the 2 Autotask webhooks into the `config.dev.yml` file
+
+### Defender Account Setup
+
+- In your [Defender account](https://defender.openzeppelin.com/), select the Hamburger icon in the upper right corner and click on **Team API Keys**
+- In the Team API Keys screen, click **Create API Key**
+- Make sure that the options for **Manage Autotasks** and **Manage Proposals and Contracts** are selected (we do not make use of the **Manage Relayers** and **Manage Sentinels** options with this template)
```

Comment:
When I only selected "Manage Autotasks" and "Manage Proposals and Contracts" I could not successfully deploy with Serverless. Here is the error output I received:
```shell
❯ serverless deploy --stage dev
Running "serverless" from node_modules
(node:66347) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
========================================================
Secrets
✔ Updated Secret (gasless_nft_minting_part_deux_FORWARDER_ADDRESS)
✔ Updated Secret (gasless_nft_minting_part_deux_NFT_ADDRESS)
Contracts
Relayers
Autotasks
✖ User is not authorized to access this resource
✖ User is not authorized to access this resource
Notifications
Environment: darwin, node 18.12.1, framework 3.27.0 (local) 3.27.0v (global), plugin 6.2.3, SDK 4.3.2
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
DefenderApiResponseError: Request failed with status code 403
    at rejectWithDefenderApiError (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-base-client/lib/api/api.js:11:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-sentinel-client/lib/api/index.js:85:20
    at async SentinelClient.apiCall (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-base-client/lib/api/client.js:26:20)
    at async DefenderDeploy.deployCategories (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-serverless/lib/cmd/deploy.js:381:31)
    at async DefenderDeploy.deploy (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-serverless/lib/cmd/deploy.js:775:9)
    at async DefenderDeploy.requestConfirmation (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/defender-serverless/lib/cmd/deploy.js:131:9)
    at async PluginManager.runHooks (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/serverless/lib/classes/plugin-manager.js:530:9)
    at async PluginManager.invoke (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/serverless/lib/classes/plugin-manager.js:564:9)
    at async PluginManager.run (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/serverless/lib/classes/plugin-manager.js:604:7)
    at async Serverless.run (/Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/serverless/lib/serverless.js:179:5)
    at async /Users/echo/Documents/openzeppelin/eth-denver-2023-templates/defender/gasless-nft-minting/node_modules/serverless/scripts/serverless.js:812:9
```

When I created an API key that allowed me to manage all of the Defender resources, the deployment worked.

Reply:
Changed it to add all permissions 9daa9144369044130f497f8c4a41219aab381ec9